<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Board Setting</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="PortComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudrateComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ConnectPushBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Status :</string>
       </property>
      </widget>
      <widget class="QLabel" name="StatusLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnected</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo 1</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="S1TotalLineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="S1CountLineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Counts</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>321</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>JOG Setting</string>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Location (0~4090) : </string>
       </property>
      </widget>
      <widget class="QLabel" name="CurLocLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pos 1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Pos1LineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pos 2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Pos2LineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="HomePushBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BwdPushBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>BWD</string>
       </property>
      </widget>
      <widget class="QPushButton" name="FwdPushBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>FWD</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="SavePushBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>450</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetupResetPushBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>450</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetupExitPushBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="SaveStatusLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo 2</string>
      </property>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="S2TotalLineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="S2CountLineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Counts</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Run</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total : </string>
      </property>
     </widget>
     <widget class="QLabel" name="S1TotalLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Completed :</string>
      </property>
     </widget>
     <widget class="QLabel" name="S1CompLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo 1</string>
      </property>
      <widget class="QPushButton" name="S1StartBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S1StopBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S1ResetBtn">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S1OnBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>On</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S1OffBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="RunExitPushBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>410</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo 2</string>
      </property>
      <widget class="QPushButton" name="S2StartBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S2StopBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S2ResetBtn">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S2OnBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>On</string>
       </property>
      </widget>
      <widget class="QPushButton" name="S2OffBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Completed :</string>
      </property>
     </widget>
     <widget class="QLabel" name="S2TotalLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="S2CompLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiona">
   <property name="text">
    <string>a</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
